docs/src/tutorials/homotopy_continuation_tutorial.md

```diff
@@ -0,0 +1,44 @@
+# Finding Steady States through Homotopy Continuation
+
+The steady states of a dynamical system ${dx \over dt} = f(x)$ can be found by solving $0 = f(x)$. This is typically a hard problem, and generally, there is no method that guarantees to find all steady states for a system that has multiple ones. However, most CRNs generate polynomial systems (the main exception is when Hill functions with non-integer exponents are used). The roots of these can reliably be found through a *homotopy continuation* algorithm. This is implemented in Julia through the [HomotopyContinuation.jl](https://www.juliahomotopycontinuation.org/) package. In this tutorial, we will demonstrate how homotopy continuation can be used to find the steady states of a CNR implemented in  Catalyst.
+
+## Basic Example 
+For this tutorial, we will use a model from the Wilhem (2009) paper (which demonstrates bistability in a small CRN). We declare the model and the parameter set for which we want to find the steady states:
+```@example hc1
+using Catalyst
+wilhelm_2009_model = @reaction_network begin
+    k1, Y --> 2X
+    k2, 2X --> X + Y
+    k3, X + Y --> Y
+    k4, X --> 0
+end k1 k2 k3 k4
+p = [:k1 => 8.0, :k2 => 2.0, :k3 => 1.0, :k4 => 1.5]
+```
+Next, we will need to extract the actual equations from our model. In addition, we will substitute in our parameter values.
+```@example hc1
+ns = convert(NonlinearSystem,wilhelm_2009_model)
+subs = Dict(Pair.(wilhelm_2009_model.ps.value,last.(p)))

```

Comment:
Maybe take a look at what I did in the workshop for handling the parameter conversion when I did the conservation law case. I think it is more in line with the public ModelingToolkit/Symbolics interface. Those fields you are accessing like `.ps` and `.ps.value` are considered private, so we should try to avoid using them in tutorials (or even within Catalyst).